.github/workflows/local-testnet.yml

```diff
@@ -40,13 +40,29 @@ jobs:
         run: make && make install-lcli
 
       - name: Start local testnet
-        run: ./start_local_testnet.sh
+        run: ./start_local_testnet.sh && sleep 60
```

Comment:
(don't mind if it stays)

Reply:
Had to dig into this again because it's been a while since I added that. I think we do need this.

If you check out the logs in some CI runs on [other branches](https://github.com/sigp/lighthouse/runs/7460369449?check_suite_focus=true) we never get past this log line, even though the `print_logs.sh` it looks like we should be dumping the whole file. 

`Jul 22 01:12:04.334 INFO Starting from known genesis state       service: beacon`

So I think what happens is we kick off all the processes and shut them down before they are fully initialized.